python/ray/air/config.py

```diff
@@ -6,6 +6,9 @@
 from ray.tune.utils.log import Verbosity
 from ray.util.annotations import PublicAPI
 
+# Move here later when ml_utils is deprecated
+from ray.util.ml_utils.checkpoint_manager import CheckpointConfig
```

Comment:
Is there a reason not to make this move now?

Reply:
circular imports

Reply:
Ah got it!